lib/datasource/git-tags/index.js

```diff
@@ -1,41 +1,38 @@
-const { getRemoteInfo } = require('isomorphic-git');
+const simpleGit = require('simple-git/promise');
 
 const cacheNamespace = 'git-tags';
 const cacheMinutes = 10;
 
+// git will prompt for known hosts or passwords, unless we activate BatchMode
+process.env.GIT_SSH_COMMAND = 'ssh -o BatchMode=yes';
```

Comment:
Is this a security hole though?

Reply:
No, with BatchMode ssh will fail if any prompt occurs, e.g. password request or when connecting to an unknown host. More specific, any ssh call will fail if a host is unknown, thus not imposing security risks. Otherwise renovate could become stuck when connecting to an unknown host.